src/containers/App/App.js

```diff
@@ -14,9 +14,6 @@ let shortID = require('short-id');
 class App extends Component {
   constructor(props) {
     super(props)
-    this.state = {
-      loading: true,
-    }
```

Comment:
I like that we were able to remove that and it still worked as intended

Reply:
Ya, I think we are going to be using the isLoading from our thunk to let our page know it should be displaying "...Loading"